specification/servicebus/resource-manager/Microsoft.ServiceBus/stable/2017-04-01/servicebus.json

```diff
@@ -4026,6 +4026,22 @@
         "forwardDeadLetteredMessagesTo": {
           "type": "string",
           "description": "Queue/Topic name to forward the Dead Letter message"
+        },
+        "clientAffine": {
+          "type": "boolean",
+          "description": "Value that indicates whether the subscription has an affinity to the client id."
+        },
+        "clientId": {
+          "type": "string",
+          "description": "Indicates the Client ID of the application that created the Subscription."
+        },
+        "durable": {
```

Comment:
The default should be true for `durable` as well as `shared`. Do you think that's fine with the current approach? i.e., in case customer doesn't pass a value for this key.

Reply:
Don't think we can specify default values here.

Client-affine default should be false, at which point the subscription is shared and durable implicitly but values of shared and durable are 'X' (don't care).

What do you think?

Reply:
we cant provide default for boolean